plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/AddToCartWithOptions.php

```diff
@@ -464,7 +471,6 @@ function ( $variation ) {
 					<input type="hidden" name="product_id" value="' . esc_attr( $product_id ) . '" />
 					<input type="hidden"
 						name="variation_id"
-						data-wp-interactive="woocommerce/add-to-cart-with-options"
```

Comment:
This element is already inside the `woocommerce/add-to-cart-with-options` store, so there is no need to declare it here too.